[2023-01-03T22:10:41.567+0100] {processor.py:153} INFO - Started process (PID=8178) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:10:41.569+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:10:41.571+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:10:41.570+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:10:41.592+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:10:41.592+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:10:41.592+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:10:41.592+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:10:41.593+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:10:41.602+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-01-03T22:11:27.189+0100] {processor.py:153} INFO - Started process (PID=8220) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:11:27.190+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:11:27.190+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:11:27.190+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:11:27.211+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:11:27.211+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:11:27.211+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:11:27.211+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:11:27.211+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:11:27.221+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-03T22:12:05.894+0100] {processor.py:153} INFO - Started process (PID=8258) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:05.895+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:12:05.896+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:05.896+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:05.913+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:05.913+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:12:05.914+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:05.914+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:12:05.914+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:05.924+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-03T22:12:45.317+0100] {processor.py:153} INFO - Started process (PID=8296) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:45.318+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:12:45.318+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:45.318+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:45.334+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:45.333+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:12:45.334+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:12:45.334+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:12:45.334+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:12:45.343+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-03T22:13:24.560+0100] {processor.py:153} INFO - Started process (PID=8334) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:13:24.561+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:13:24.562+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:13:24.562+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:13:24.577+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:13:24.577+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:13:24.577+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:13:24.577+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:13:24.578+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:13:24.587+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-03T22:14:03.277+0100] {processor.py:153} INFO - Started process (PID=8372) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:03.278+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:14:03.279+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:03.279+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:03.298+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:03.298+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:14:03.298+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:03.298+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:14:03.299+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:03.308+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-03T22:14:42.517+0100] {processor.py:153} INFO - Started process (PID=8410) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:42.518+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:14:42.519+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:42.519+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:42.536+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:42.535+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:14:42.536+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:14:42.536+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:14:42.536+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:14:42.546+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-03T22:15:21.549+0100] {processor.py:153} INFO - Started process (PID=8449) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:15:21.550+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:15:21.552+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:15:21.552+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:15:21.566+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:15:21.566+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:15:21.566+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:15:21.566+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:15:21.566+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:15:21.579+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-03T22:31:15.812+0100] {processor.py:153} INFO - Started process (PID=8539) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:15.814+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:31:15.815+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:15.815+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:15.832+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:15.831+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:31:15.833+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:15.833+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:31:15.833+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:15.843+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-03T22:31:55.728+0100] {processor.py:153} INFO - Started process (PID=8583) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:55.729+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:31:55.729+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:55.729+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:55.746+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:55.746+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:31:55.746+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:31:55.746+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:31:55.746+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:31:55.755+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-03T22:32:34.303+0100] {processor.py:153} INFO - Started process (PID=8621) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:32:34.304+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:32:34.305+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:32:34.305+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:32:34.319+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:32:34.319+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:32:34.319+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:32:34.319+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:32:34.319+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:32:34.327+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-03T22:33:14.333+0100] {processor.py:153} INFO - Started process (PID=8659) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:14.334+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:33:14.335+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:14.335+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:14.354+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:14.353+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:33:14.354+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:14.354+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:33:14.354+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:14.364+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-03T22:33:52.816+0100] {processor.py:153} INFO - Started process (PID=8697) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:52.817+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:33:52.818+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:52.818+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:52.835+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:52.835+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:33:52.835+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:33:52.835+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:33:52.836+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:33:52.846+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-03T22:34:32.759+0100] {processor.py:153} INFO - Started process (PID=8735) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:34:32.760+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:34:32.760+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:34:32.760+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:34:32.775+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:34:32.775+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:34:32.775+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:34:32.775+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:34:32.775+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:34:32.784+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:35:12.342+0100] {processor.py:153} INFO - Started process (PID=8773) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:12.343+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:35:12.344+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:12.344+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:12.358+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:12.358+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:35:12.358+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:12.358+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:35:12.358+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:12.366+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:35:50.497+0100] {processor.py:153} INFO - Started process (PID=8811) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:50.499+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:35:50.499+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:50.499+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:50.513+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:50.513+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:35:50.514+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:35:50.514+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:35:50.514+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:35:50.522+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:36:29.787+0100] {processor.py:153} INFO - Started process (PID=8849) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:36:29.788+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:36:29.789+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:36:29.789+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:36:29.803+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:36:29.803+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:36:29.803+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:36:29.803+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:36:29.803+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:36:29.812+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:37:07.640+0100] {processor.py:153} INFO - Started process (PID=8887) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:07.641+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:37:07.642+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:07.642+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:07.656+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:07.656+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:37:07.657+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:07.657+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:37:07.657+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:07.666+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:37:45.402+0100] {processor.py:153} INFO - Started process (PID=8925) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:45.403+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:37:45.403+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:45.403+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:45.417+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:45.417+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:37:45.418+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:37:45.418+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:37:45.418+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:37:45.426+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:38:24.358+0100] {processor.py:153} INFO - Started process (PID=8963) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:38:24.360+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:38:24.361+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:38:24.361+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:38:24.395+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:38:24.394+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:38:24.395+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:38:24.395+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:38:24.395+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:38:24.410+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2023-01-03T22:39:03.767+0100] {processor.py:153} INFO - Started process (PID=9001) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:03.768+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:39:03.769+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:03.769+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:03.785+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:03.784+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:39:03.786+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:03.785+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:39:03.786+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:03.795+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-03T22:39:43.091+0100] {processor.py:153} INFO - Started process (PID=9039) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:43.092+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:39:43.093+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:43.093+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:43.107+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:43.107+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:39:43.108+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:39:43.108+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:39:43.108+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:39:43.116+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:40:22.074+0100] {processor.py:153} INFO - Started process (PID=9077) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:40:22.075+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:40:22.075+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:40:22.075+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:40:22.089+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:40:22.089+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:40:22.089+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:40:22.089+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:40:22.090+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:40:22.098+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:41:00.821+0100] {processor.py:153} INFO - Started process (PID=9115) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:00.822+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:41:00.823+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:00.823+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:00.838+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:00.838+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:41:00.839+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:00.838+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:41:00.839+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:00.853+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-03T22:41:38.114+0100] {processor.py:153} INFO - Started process (PID=9153) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:38.115+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:41:38.115+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:38.115+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:38.130+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:38.129+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:41:38.130+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:41:38.130+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:41:38.130+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:41:38.140+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-03T22:42:16.710+0100] {processor.py:153} INFO - Started process (PID=9193) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:16.712+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:42:16.712+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:16.712+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:16.731+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:16.731+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:42:16.731+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:16.731+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:42:16.731+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:16.743+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-01-03T22:42:55.425+0100] {processor.py:153} INFO - Started process (PID=9233) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:55.426+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:42:55.426+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:55.426+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:55.444+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:55.443+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:42:55.444+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:42:55.444+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:42:55.444+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:42:55.452+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-03T22:43:35.825+0100] {processor.py:153} INFO - Started process (PID=9271) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:43:35.826+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:43:35.826+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:43:35.826+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:43:35.841+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:43:35.841+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:43:35.841+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:43:35.841+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:43:35.842+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:43:35.850+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:44:13.671+0100] {processor.py:153} INFO - Started process (PID=9309) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:13.672+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:44:13.672+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:13.672+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:13.686+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:13.686+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:44:13.686+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:13.686+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:44:13.687+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:13.695+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:44:52.327+0100] {processor.py:153} INFO - Started process (PID=9347) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:52.328+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:44:52.328+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:52.328+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:52.343+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:52.342+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:44:52.343+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:44:52.343+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:44:52.343+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:44:52.351+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:45:32.038+0100] {processor.py:153} INFO - Started process (PID=9385) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:45:32.039+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:45:32.040+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:45:32.040+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:45:32.053+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:45:32.053+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:45:32.054+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:45:32.054+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:45:32.054+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:45:32.062+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:46:10.721+0100] {processor.py:153} INFO - Started process (PID=9423) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:10.722+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:46:10.723+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:10.723+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:10.737+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:10.737+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:46:10.738+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:10.737+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:46:10.738+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:10.746+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:46:46.293+0100] {processor.py:153} INFO - Started process (PID=9461) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:46.294+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:46:46.295+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:46.295+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:46.309+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:46.309+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:46:46.309+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:46:46.309+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:46:46.309+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:46:46.317+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-03T22:47:26.257+0100] {processor.py:153} INFO - Started process (PID=9499) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:47:26.258+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:47:26.259+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:47:26.258+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:47:26.273+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:47:26.273+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:47:26.274+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:47:26.274+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:47:26.274+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:47:26.282+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:48:04.505+0100] {processor.py:153} INFO - Started process (PID=9537) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:04.507+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:48:04.507+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:04.507+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:04.528+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:04.527+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:48:04.528+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:04.528+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:48:04.528+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:04.542+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-01-03T22:48:45.022+0100] {processor.py:153} INFO - Started process (PID=9575) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:45.023+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:48:45.023+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:45.023+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:45.037+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:45.037+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:48:45.038+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:48:45.037+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:48:45.038+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:48:45.051+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-03T22:49:26.227+0100] {processor.py:153} INFO - Started process (PID=9613) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:49:26.228+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:49:26.228+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:49:26.228+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:49:26.242+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:49:26.242+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:49:26.242+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:49:26.242+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:49:26.243+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:49:26.251+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:50:05.258+0100] {processor.py:153} INFO - Started process (PID=9651) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:05.259+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:50:05.260+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:05.259+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:05.274+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:05.273+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:50:05.274+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:05.274+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:50:05.274+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:05.282+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:50:45.240+0100] {processor.py:153} INFO - Started process (PID=9689) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:45.241+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:50:45.242+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:45.241+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:45.255+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:45.255+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:50:45.256+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:50:45.256+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:50:45.256+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:50:45.264+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:51:23.442+0100] {processor.py:153} INFO - Started process (PID=9727) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:51:23.443+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:51:23.444+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:51:23.444+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:51:23.458+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:51:23.458+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:51:23.459+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:51:23.458+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:51:23.459+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:51:23.467+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:52:02.663+0100] {processor.py:153} INFO - Started process (PID=9765) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:02.663+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:52:02.664+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:02.664+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:02.678+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:02.678+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:52:02.678+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:02.678+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:52:02.678+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:02.687+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-03T22:52:44.841+0100] {processor.py:153} INFO - Started process (PID=9803) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:44.842+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:52:44.842+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:44.842+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:44.859+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:44.859+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:52:44.859+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:52:44.859+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:52:44.860+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:52:44.869+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-03T22:53:23.072+0100] {processor.py:153} INFO - Started process (PID=9841) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:53:23.073+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:53:23.074+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:53:23.074+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:53:23.089+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:53:23.089+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:53:23.089+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:53:23.089+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:53:23.089+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:53:23.098+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-03T22:54:04.346+0100] {processor.py:153} INFO - Started process (PID=9884) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:04.349+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:54:04.350+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:04.350+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:04.382+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:04.381+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:54:04.383+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:04.382+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:54:04.384+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:04.403+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2023-01-03T22:54:47.472+0100] {processor.py:153} INFO - Started process (PID=9924) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:47.473+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:54:47.474+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:47.474+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:47.493+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:47.492+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:54:47.493+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:54:47.493+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:54:47.494+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:54:47.505+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-01-03T22:55:30.003+0100] {processor.py:153} INFO - Started process (PID=9962) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:55:30.017+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:55:30.017+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:55:30.017+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:55:30.042+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:55:30.041+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:55:30.042+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:55:30.042+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:55:30.043+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:55:30.056+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2023-01-03T22:56:12.887+0100] {processor.py:153} INFO - Started process (PID=10005) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:12.888+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:56:12.888+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:12.888+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:12.915+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:12.914+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:56:12.915+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:12.915+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:56:12.916+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:12.929+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-01-03T22:56:54.515+0100] {processor.py:153} INFO - Started process (PID=10044) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:54.517+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-03T22:56:54.518+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:54.518+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:54.546+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:54.545+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-03T22:56:54.546+0100] {logging_mixin.py:137} INFO - [2023-01-03T22:56:54.546+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-03T22:56:54.546+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-03T22:56:54.563+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
